scripts/docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Basic dependencies
+ENV HOME /opt
+RUN apt-get update && apt-get install -y apt-utils curl git git-core bsdmainutils
+
+# Install python dependencies
+ENV PYENV_ROOT $HOME/pyenv
+ENV PATH $PYENV_ROOT/bin:$PATH
+RUN apt-get install -y curl gcc make zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev libssl-dev libffi-dev
+RUN git clone https://github.com/pyenv/pyenv.git $HOME/pyenv
+
+RUN eval "$(pyenv init -)" && \
+    pyenv install 3.7.1 && \
+    pyenv global 3.7.1 && \
+    pip install --upgrade pip && \
+    pyenv rehash
+ENV PATH=$PYENV_ROOT/shims
+
+RUN pip3 install git+https://github.com/algorand/py-algorand-sdk/ -q \
+    && pip3 install behave -q
+
+# Install Javascript dependencies
+RUN curl -sL https://deb.nodesource.com/setup_13.x | bash -
+RUN apt-get install -y nodejs
+
+# Install Java dependencies
+RUN apt-get install -y maven default-jdk
+
+# Install Go dependencies
```

Comment:
Why not use the golang official image as a base instead of installing all this on ubuntu?

Reply:
I thought about that, but you could say the same about java/javascript/python. In terms of complexity python is the most involved installation at the moment.